src/scripts/components/Tooltip.jsx

```diff
@@ -195,7 +171,19 @@ class Tooltip extends React.Component {
    * @return {void}
    */
   openTooltip(tooltip) {
-    tooltip.timeline.play();
+    const { delay } = this.props;
+
+    // Convert delay to a number
+    // Takes care of multiple PropTypes
+    const delayAmount = Number.isInteger(delay) ? Number(delay) : 1000;
```

Comment:
Consider calling this variable `duration` or `delayDuration` versus `amount`. It's a personal preference, so no worries if you like `amount` better.

Reply:
👍 yeah that makes sense